paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/lookup/FileStoreLookupFunction.java

```diff
@@ -183,6 +180,9 @@ private void open() throws Exception {
                                     path,
                                     joinKeys,
                                     getRequireCachedBucketIds());
+                    LOG.info(
+                            "Created lookup table for {}. Type is PrimaryKeyPartialLookupTable with LocalQueryExecutor.",
+                            table.name());
                 } catch (UnsupportedOperationException ignore2) {
                 }
             }
```

Comment:
Logging the reason why it is not supported. (Cus table bucket mode is not HASH_FIXED).